source/crud/tests/unified/create-id-types.yml

```diff
@@ -0,0 +1,903 @@
+description: CRUD ID Type Tests
+schemaVersion: '1.0'
+createEntities:
+  - client:
+      id: client
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: database
+      client: client
+      databaseName: crud_id
+  - collection:
+      id: collection
+      database: database
+      collectionName: type_tests
+tests:
+  - description: inserting _id with type double via insertOne
+    operations:
+      - {name: dropCollection, object: database, arguments: {collection: type_tests}}
```

Comment:
Drop database and collection are expensive, slow commands. If we're adding so many tests it would be good to avoid it.

Also do we really need to test every combination of operation and BSON type?

Reply:
Perhaps it would be faster to put all the operations in one test. There's a lot of overhead to setting up a unified test which really adds up in runtime cost when adding 150 tests.